pkg/controller/plan/kubevirt.go

```diff
@@ -601,8 +601,12 @@ func (r *KubeVirt) createVolumesForOvirt(vm ref.Ref) (pvcNames []string, err err
 		return
 	}
 
-	storageName := &r.Context.Map.Storage.Spec.Map[0].Destination.StorageClass
 	for _, da := range ovirtVm.DiskAttachments {
+		if da.Disk.StorageType == "lun" {
+			continue
+		}
+		// Set storage map read here in order to prevent nil pointer with LUN disk.
```

Comment:
now that comment is more complex to understand, how about
```suggestion
		// The VM has a disk image so the storage map is necessarily not empty and we can read the storage class from it
```
and this makes me wonder - don't we need to take the destination storage class that is set for the storage domain that the da.Disk resides in?

Reply:
for lun we don't use storage class it's just empty string. i'll rephrase the comment as you suggested.

Reply:
> for lun we don't use storage class it's just an empty string.

right, I meant for non-lun disks - I'm missing something there, I'd expect that for each disk we'll check its storage domain and get the corresponding destination from the map, but maybe it's done differently

Reply:
i agree. something is strange here. it worked before, right? if not there is a bug here. if a VM has 2 disks on different SDs on source, mapped to different storage classes.

Reply:
maybe that's the root cause for https://issues.redhat.com/browse/MTV-445

Reply:
i don't think so. the problem that it may cause is only if the source storage is in multiple places and the destination as well. it shouldn't break the migration, just use the wrong SC for one (or more) disks.

Reply:
hard to tell without investigating that issue but I commented on https://issues.redhat.com/browse/MTV-445 so we'll keep this in mind

Reply:
@liranr23 shall we handle it separately?

Reply:
ack, filed https://github.com/kubev2v/forklift/issues/427